vtkext/private/module/vtkF3DUIActor.h

```diff
@@ -20,7 +21,18 @@ class vtkF3DUIActor : public vtkProp
   static vtkF3DUIActor* New();
   vtkTypeMacro(vtkF3DUIActor, vtkProp);
 
-  using CheatSheetTuple = std::tuple<std::string, std::string, std::string>;
+  /**
+   * Enumeration of available cheatsheet binding types.
```

Comment:
mention its a duplicate with `library/public/interactor.h` BindingType. Mention the duplication in interactor.h as well.